examples/vpc-endpoints/outputs.tf

```
@@ -0,0 +1,23 @@
+output "public_subnet_cidrs" {
+  value = module.subnets.public_subnet_cidrs
+}
+
+output "private_subnet_cidrs" {
+  value = module.subnets.private_subnet_cidrs
+}
+
+output "vpc_cidr" {
+  value = module.vpc.vpc_cidr_block
+}
+
+output "vpc_id" {
+  value = module.vpc.vpc_id
+}
+
+output "gateway_vpc_endpoints" {
+  value = module.vpc-endpoints.gateway_vpc_endpoints
```

```suggestion
  value = module.vpc_endpoints.gateway_vpc_endpoints
```

Let's be consistent and use `_` in all resource names